### PR TITLE
v21.2.3 Release Notes

### PR DESCRIPTION
**Pull request by lnhsingh**
Closes https://cockroachlabs.atlassian.net/browse/DOC-1890

### CODE REVIEW

File: _config_base.yml [hunk 1]
```diff
@@ -84,11 +84,11 @@ release_info:
     start_time: 2021-11-15 11:01:26.34274101 +0000 UTC
     version: v21.1.12
   v21.2:
-    build_time: 2021/12/01 11:00:26 (go1.16.6)
+    build_time: 2021/12/13 11:00:26 (go1.16.6)
```
Comment (stbof) [inline _config_base.yml:87]: If these are being published tomorrow all dates need to change to 12/14.

File: releases/v21.2.3.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+---
+title: What&#39;s New in v21.2.3
+toc: true
+summary: Additions and changes in CockroachDB version v21.2.3 since version v21.2.2
+---
+
+## December 13, 2021
```
Comment (stbof) [inline releases/v21.2.3.md:7]:
```suggestion
## December 14, 2021
```

File: releases/v21.2.3.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    EXPERIMENTAL SCRUB TABLE t WITH OPTIONS INDEX ALL;
+    ~~~
+
+    [#71595][#71595]
+
+- [`RESTORE TABLE`](../v21.2/restore.html) for a regional by row table into a multiregion database with the same regions as the backed up database is now allowed. The user must ensure that the regions in the backed up database and the database being restored into match, and are added in the same order, for the [`RESTORE`](../v21.2/restore.html) to work. [#72088][#72088]
+- The structured payloads used for telemetry logs now include two new fields: `CostEstimate` and `Distribution`. `CostEstimate` is the cost of the query as estimated by the [optimizer](../v21.2/cost-based-optimizer.html), and `Distribution` is the distribution of the DistSQL query plan (local, full, or partial). [#73410][#73410]
+- Fixed a bug which allowed [computed columns](../v21.2/computed-columns.html) to also have [`DEFAULT`](../v21.2/default-value.html) expressions. [#73190][#73190]
+
+### DB Console changes
```
Comment (stbof) [inline releases/v21.2.3.md:86]: I wonder if this heading should be DB Console and Cloud Console. (For the 2nd bullet).

File: releases/v21.2.3.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- The absolute links on the [**Advanced Debug** page](../v21.2/ui-debug-pages.html) in DB Console have been updated to relative links. This will enable these links to work with the superuser dashboard in [Cloud Console](https://cockroachlabs.cloud). [#73067][#73067]
+- When an error is encountered in the [**Statements**](../v21.2/ui-statements-page.html), [**Transactions**](../v21.2/ui-transactions-page.html), or [**Sessions**](../v21.2/ui-sessions-page.html) page, the user can now click on a reload button to reload the page. [#73115][#73115]
+
+### Bug fixes
+
+- Fixed a bug where [`GENERATED ... IDENTITY`](../v21.2/create-table.html#identity-columns) would panic when using a non-`INT` value during table creation. [#73029][#73029]
+- Fixed a bug whereby setting the `CACHE` for a sequence to `1` was ignored. Before this change [`ALTER SEQUENCE ... CACHE 1`](../v21.2/alter-sequence.html) would succeed but would not modify the cache value. [#71449][#71449]
+- Fixed a very unlikely bug where a crash during [startup](../v21.2/cockroach-start.html) may cause all subsequent starts to fail. [#73124][#73124]
```
Comment (stbof) [inline releases/v21.2.3.md:96]: I don't think we should include `very unlikely`.  It's not really helpful.

**Review by stbof — CHANGES_REQUESTED** [review]
See comments.

File: releases/v21.2.3.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
+---
+title: What&#39;s New in v21.2.3
+toc: true
+summary: Additions and changes in CockroachDB version v21.2.3 since version v21.2.2
```
Comment (stbof) [inline releases/v21.2.3.md:4]: I wonder if we need `version` since it already has v21.2.3, etc.

**Review by lnhsingh — COMMENTED** [review]
Thanks for the review!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12539#-:-Mqp7d_Y5KTQDFyTEwx_:b-9qdoz3)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stbof)

___
*[_config_base.yml, line 87 at r1](https://reviewable.io/reviews/cockroachdb/docs/12539#-MqVoQnKtxHNBXdg4nV5-r1-87:-Mqp7Pq07l7My0Q2sOwp:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/fa0dbfcb63902d1cc8d9db31c10e69ab98604bca/_config_base.yml#L87)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

If these are being published tomorrow all dates need to change to 12/14.
</blockquote></details>

Done.
___
*[releases/v21.2.3.md, line 4 at r1](https://reviewable.io/reviews/cockroachdb/docs/12539#-MqVoQnKtxHNBXdg4nV8-r1-4:-Mqp7B3VDp7ZIThQqMJM:bss4a2b) ([raw file](https://github.com/cockroachdb/docs/blob/fa0dbfcb63902d1cc8d9db31c10e69ab98604bca/releases/v21.2.3.md#L4)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

I wonder if we need `version` since it already has `v21.2.3`, where `v` stands for version, doesn't it?
</blockquote></details>

Removed.
___
*[releases/v21.2.3.md, line 7 at r1](https://reviewable.io/reviews/cockroachdb/docs/12539#-MqVoQnKtxHNBXdg4nV8-r1-7:-Mqp7ItDARqxXaq4hPAF:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/fa0dbfcb63902d1cc8d9db31c10e69ab98604bca/releases/v21.2.3.md#L7)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

```suggestion
## December 14, 2021
```
</blockquote></details>

Done.
___
*[releases/v21.2.3.md, line 86 at r1](https://reviewable.io/reviews/cockroachdb/docs/12539#-MqVoQnKtxHNBXdg4nV8-r1-86:-Mqp6rpZ9CRxFZdvSh7a:bv54h2c) ([raw file](https://github.com/cockroachdb/docs/blob/fa0dbfcb63902d1cc8d9db31c10e69ab98604bca/releases/v21.2.3.md#L86)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

I wonder if this heading should be DB Console and Cloud Console (or simply Console changes). (For the 2nd bullet).
</blockquote></details>

I'm inclined to leave it as for now since it's all changes in the DB Console (even if the changes are enabling something to work in the Cloud Console). Changes made to the Cloud Console are outlined in Cloud Release Notes
___
*[releases/v21.2.3.md, line 96 at r1](https://reviewable.io/reviews/cockroachdb/docs/12539#-MqVoQnKtxHNBXdg4nV8-r1-96:-Mqp7YEYEbkA26vVHvWf:bss4a2b) ([raw file](https://github.com/cockroachdb/docs/blob/fa0dbfcb63902d1cc8d9db31c10e69ab98604bca/releases/v21.2.3.md#L96)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

I don't think we should include `very unlikely`.  It's not really helpful.
</blockquote></details>

Removed.

**Review by stbof — APPROVED** [review]

**Pull request merged**